packages/hub/process-controllers/hub-event-listener-controller.ts

```diff
@@ -0,0 +1,43 @@
+import { ContractSubscriptionEventHandler } from '../services/contract-subscription-event-handler';
+import { runInitializers, createRegistry } from '../main';
+import * as Sentry from '@sentry/node';
+import { Registry, Container } from '@cardstack/di';
+import { eventListenerLog } from '../utils/logger';
+
+export class HubEventListenerController {
+  logger = eventListenerLog;
+  static logger = eventListenerLog;
+
+  static async create(serverConfig?: {
+    registryCallback?: (r: Registry) => void;
+  }): Promise<HubEventListenerController> {
+    this.logger.info(`booting pid:${process.pid}`);
+    runInitializers();
+
+    let registry = createRegistry();
+    if (serverConfig?.registryCallback) {
+      serverConfig.registryCallback(registry);
+    }
+    let container = new Container(registry);
```

Comment:
this is a smell--we should not create containers at this level. the hob bot controller and the worker controller are actually incorrect in this regards